src/include/misc.h

```diff
@@ -38,6 +38,7 @@
 #define WT_DIVIDER "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
 
 /* Basic constants. */
+#define WT_HUNDRED (1000)
```

Comment:
This is wrong and should be removed. For something less than 1000 I think we don't need this constant.

Reply:
Fixed in 0e6b43406c158379072274241f2c15de26c317b5